docs/pages/reference/metrics.mdx

```diff
@@ -2,12 +2,38 @@
 title: Teleport Metrics
 description: Comprehensive list of all metrics exposed by Teleport.
 ---
-The following metrics are available:
 
 <Notice scope={["cloud"]} type="tip">
 
 Teleport Cloud does not expose monitoring endpoints for the Auth Service and Proxy Service.
 
 </Notice>
 
+Teleport metrics data is primarily used for performance monitoring. If you'd like to
+monitor Teleport usage, consider utilizing our *fluentd* plugin to push Audit Events into your
```

Comment:
```suggestion
monitor Teleport usage, consider utilizing our Event Handler plugin to push Audit Events into your
```
While the Event Handler was designed with Fluentd in mind, I think we should use a more general title so we don't put off users who are using Logstash, a Splunk forwarder, or another solution to forward audit logs, since the Event Handler can work with these, too (albeit with some extra steps, e.g., in the [Elastic Stack guide](https://goteleport.com/docs/management/guides/elastic-stack/)).